web/themes/custom/pineapple/sass/base/_base.scss

```diff
@@ -0,0 +1,20 @@
+.block-system-main-block {
+ width: $gutter-size;
+  margin: 0 auto;
```

Comment:
This is not the correct place or way to add gutters on your site, look at the `container` class on the uwl site: https://www.uwl.ac.uk/

Reply:
done